rebar.config

```diff
@@ -19,4 +19,9 @@
     {source_url, "https://github.com/certifi/erlang-certifi"},
     {assets, "assets"},
     {api_reference, true}
-]}.
\ No newline at end of file
+]}.
+
+%% Since OTP 26
+{dialyzer, [
+    {plt_extra_apps, [public_key]}
+]}.
```

Comment:
```
Run rebar3 dialyzer
===> Verifying dependencies...
===> Analyzing applications...
===> Compiling certifi
===> Dialyzer starting, this may take a while...
===> Add debug_info to compiler options (erl_opts) if Dialyzer fails to load Core Erlang.
===> Updating plt...
===> Resolving project files...
===> Updating base plt...
===> Resolving base files...
===> Building with 212 files in /home/runner/.cache/rebar3/rebar3_2[6](https://github.com/kianmeng/erlang-certifi/actions/runs/9627560355/job/26554777720#step:6:7).2.5_plt...
===> Copying /home/runner/.cache/rebar3/rebar3_26.2.5_plt to /home/runner/work/erlang-certifi/erlang-certifi/_build/default/rebar3_26.2.5_plt...
===> Checking 212 files in _build/default/rebar3_26.2.5_plt...
===> Doing success typing analysis...
===> Resolving project warning files...
===> Analyzing 2 files with _build/default/rebar3_26.2.5_plt...

./certifi_pt.erl
Line [9](https://github.com/kianmeng/erlang-certifi/actions/runs/9627560355/job/26554777720#step:6:10) Column 10: Unknown function public_key:pem_decode/1
===> Warnings written to _build/default/26.2.5.dialyzer_warnings
===> Warnings occurred running dialyzer: 1
Error: Process completed with exit code 1.
```